clang/test/PCH/cxx2a-constraints-crash.cpp

```diff
@@ -27,3 +25,12 @@ int main() {
 }
 
 #endif
+
+namespace GH99036 {
+
+template <typename T>
+concept C; // expected-error {{expected '='}}
+// expected-note@32 {{declared here}}
```

Comment:
Should have used `@-1` here.

Reply:
Yep, you can also arrange them in the following form:

// expected-error@-1 {{…}} \
// expected-note@-1 {{…}}

(note the backslash)

Which is a bit tidier.

Reply:
Done. Will push tomorrow or if additional changes are requested.

```cpp
template <typename T>
concept C;
// expected-error@-1 {{expected '='}} \
// expected-note@-1 {{declared here}}

template <C U> void f();
// expected-error@-1 {{a concept definition cannot refer to itself}}
```